cocos/base/ccUTF8.h

```diff
@@ -196,7 +196,7 @@ class CC_DLL StringUTF8
     struct CharUTF8
     {
         std::string _char;
-        bool isAnsi() { return _char.size() == 1; }
+        bool isAnsi() const { return _char.size() == 1; }
```

Comment:
ut8 chars' size may be 1 too, right?

Reply:
chars' size is 1 for ansi char, maybe great than 1 other cases

Reply:
Yep, what i mean is that

* size > 1, not ansi
* size = 1, may be ansi or not

so can not use it to determine if it is ansi or not

Reply:
so the function name is not accurate. em, this is the old code,
could you give some advice about it?

Reply:
I didn't notice that it is old code. I have not idea too.

Reply:
StringUTF8 it my api, for ease work cursor in TextField and other.
See specification UTF8 https://tools.ietf.org/html/rfc3629
UTF-8, the object of this memo, has a one-octet encoding unit.  It
   uses all bits of an octet, but has the quality of preserving the full
   US-ASCII [US-ASCII] range: US-ASCII characters are encoded in one
   octet having the normal US-ASCII value, and any octet with such a
   value can only stand for a US-ASCIIUTF-8, the object of this memo, has a one-octet encoding unit.  It
   uses all bits of an octet, but has the quality of preserving the full
   US-ASCII [US-ASCII] range: US-ASCII characters are encoded in one
   octet having the normal US-ASCII value, and any octet with such a
   value can only stand for a US-ASCII character, and nothing else. character, and nothing else.

Right name it function - isASCII.
In Russian ASCII encoding for Window named alias - ansi.

Sorry for the confusion.
Possible name:
isASCII
isOneByteChar
isOneByte